coalahtml/coala_html.py

```diff
@@ -35,7 +35,7 @@ def main():
 
     dir_path = create_dir(os.path.abspath(args.dir))
 
-    if len(list(os.walk(dir_path))) > 0:
+    if len(list(os.walk(dir_path))) > 0:  # pragma: no cover
```

Comment:
Basically, here you're testing if it's a directory or a file, right? If yes, then use respective methods, like `os.is_file` or `os.is_dir`, instead.

Also, why not cover this test case?